code/game/objects/items/tools/maintenance_tools.dm

```diff
@@ -463,3 +463,86 @@
 	icon_state = "marine_flamerpack"
 	w_class = WEIGHT_CLASS_BULKY
 	max_fuel = 500 //Because the marine backpack can carry 260, and still allows you to take items, there should be a reason to still use this one.
+
+/obj/item/tool/handheld_charger
+	name = "handheld charger"
+	desc = "A hand-held, lightweight cell charger. It isn't going to give you tons of power, but it can help in a pinch."
+	icon = 'icons/obj/items/items.dmi'
+	icon_state = "handheldcharger_empty"
+	item_state = "handheldcharger_empty"
+	w_class = WEIGHT_CLASS_SMALL
+	flags_atom = CONDUCT
+	force = 6.0
+	throw_speed = 2
+	throw_range = 9
+	flags_equip_slot = ITEM_SLOT_BELT
+	materials = list(/datum/material/metal = 50, /datum/material/glass = 20)
+	/// This is the cell we ar charging
+	var/obj/item/cell/cell
+	var/recharging = FALSE
+
+/obj/item/tool/handheld_charger/Initialize()
+	. = ..()
+	cell = null
+
+/obj/item/tool/handheld_charger/attack_self(mob/user)
+	if(!cell)
+		to_chat(user, span_notice("You need some cell to be useful, idiot"))
+		return
+
+	if(recharging)///Already using it.
+		recharging = FALSE
+		to_chat(user, span_notice("You stop using the recharger."))
+		return
+
+	if(cell.charge >= cell.maxcharge)
+		to_chat(user, span_notice("\The [cell] is already fully charged."))
+		return
+	recharging = TRUE
+	while(do_after(user, 1 SECONDS, TRUE, src, BUSY_ICON_GENERIC))
+		cell.charge += 200
```

Comment:
Make sure this cant go over cell.maxcharge by using a min() check

cell.charge = min(cell.charge + 200, cell.maxcharge)

Reply:
Implemented. Thanks